extensions/docs/msal-node-extensions.md

```diff
@@ -109,9 +166,61 @@ const publicClientConfig = {
 const pca = new PublicClientApplication(publicClientConfig);
 ```
 
+Example (for Electron node-js desktop app):-
+
+authConfig.js:-
+```js
+const AAD_ENDPOINT_HOST = "https://login.microsoftonline.com/"; // include the trailing slash
+const REDIRECT_URI = "ENTER_REDIRECT_URI";
+
+const cachePath = "path/to/cache/file.json";
+
+/*define persistence config based on the appropriate persistence you are using(e.g- FilePersistenceWithDataProtection, generic PersistenceCreateor, etc)*/
+
+//defining persistence config for PersistenceCreator
+const persistenceConfiguration = {
+    cachePath,
+    dataProtectionScope: DataProtectionScope.CurrentUser,
+    serviceName: "test-msal-electron-service",
+    accountName: "test-msal-electron-account",
+    usePlaintextFileOnLinux: false,
+}
+
+  const msalConfig = {
+    auth: {
+        clientId: "CLIENT_ID_HERE",
+        authority: `${AAD_ENDPOINT_HOST}TENANT_ID_HERE`,
+    },
+    cache: {
+        cachePlugin: null // set later in main.js as shown above 
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback(loglevel, message, containsPii) {
+                console.log(message);
+            },
+            piiLoggingEnabled: false,
+            logLevel: LogLevel.Verbose,
+        },
+    },
+};
+...
+
+module.exports = {
+  msalConfig: msalConfig,
+  protectedResources: protectedResources,
+  REDIRECT_URI: REDIRECT_URI,
+  persistenceConfiguration
+};
+
+```
+
 Note that MSAL will not read and write to persistence by default. You will have to call PublicClientApplication.tokenCache.readFromPersistence() and PublicClientApplication.tokenCache.writeToPersistence() anytime you trigger and MSAL Node operation that alters the token cache.
```

Comment:
> Note that MSAL will not read and write to persistence by default. You will have to call PublicClientApplication.tokenCache.readFromPersistence() and PublicClientApplication.tokenCache.writeToPersistence() anytime you trigger and MSAL Node operation that alters the token cache.

Is this true? cc @sameerag 
I think cache read/write will be triggered whenever a cache API is used e.g. `cache.getAllAccounts()`
Or am I missing the intent here?

Reply:
Not true. I think it should get triggered when we set `allowPeristence: true` in config and enable the wrapper on initial pca construction. Example [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/extensions/samples/msal-node-extensions/index.js).

Reply:
removed